src/styles/blocks/normalize.scss

```
@@ -0,0 +1,10 @@
+.page {
+  font-family: Arial, sans-serif;
+
+  &__body {
+    display: flex;
+    justify-content: center;
+    margin: 50px 0;
+    max-width: 100%;
```

can it be more than 100%
```suggestion
    width: 100%;
```